content/docs/rfcs/13/README.md

```diff
@@ -81,31 +96,49 @@ To perform pagination, each `WakuMessage` stored at a node running the `WakuStor
-  - By deploying secure 2-party computations in which the querying node obtains the historical messages of a certain topic whereas the queried node learns nothing about the query. Examples of such 2PC protocols are secure one-way Private Set Intersections (PSI). <!-- TODO: add a reference for PSIs? --> <!-- TODO: more techniques to be included -->. 
+- **Anonymous query**: This feature guarantees that nodes can anonymously query historical messages from other nodes i.e., without disclosing the exact topics of waku messages they are interested in.  
+As such, no adversary in the `WakuStore` protocol would be able to learn which peer is interested in which content filters i.e., content topics of waku message. 
+The current version of the `WakuStore` protocol does not provide anonymity for historical queries as the querying node needs to directly connect to another node in the `WakuStore` protocol and explicitly disclose the content filters of its interest to retrieve the corresponding messages. 
+However, one can consider preserving anonymity through one of the following ways: 
+  - By hiding the source of the request i.e., anonymous communication. That is the querying node shall hide all its PII in its history request e.g., its IP address.
+  This can happen by the utilization of a proxy server or by using Tor. 
+  Note that the current structure of historical requests does not embody any piece of PII, otherwise, such data fields must be treated carefully to achieve query anonymity. 
+  <!-- TODO: if nodes have to disclose their PeerIDs (e.g., for authentication purposes) when connecting to other nodes in the store protocol, then Tor does not preserve anonymity since it only helps in hiding the IP. So, the PeerId usage in switches must be investigated further. Depending on how PeerId is used, one may be able to link between a querying node and its queried topics despite hiding the IP address--> 
+  - By deploying secure 2-party computations in which the querying node obtains the historical messages of a certain topic whereas the queried node learns nothing about the query. 
+  Examples of such 2PC protocols are secure one-way Private Set Intersections (PSI). 
+  <!-- TODO: add a reference for PSIs? --> <!-- TODO: more techniques to be included --> 
 <!-- TODO: Censorship resistant: this is about a node that hides the historical messages from other nodes. This attack is not included in the specs since it does not fit the passive adversarial model (the attacker needs to deviate from the store protocol).-->
 
 # Changelog
 
 ### Next 
 - Added the initial threat model and security analysis.
+- Replaced the `topics` field of `HistoryQuery` with a newly defined message type `HistoryContentFilter`.
```

Comment:
Here is a changelog guideline that I highly recommend to help with consistency across the code base https://keepachangelog.com/en/1.0.0/

Reply:
Great, thanks!

Reply:
Addressed.